api/src/pcapi/routes/backoffice/non_payment_notices/non_payment_notices_blueprint.py

```diff
@@ -0,0 +1,115 @@
+import sqlalchemy as sa
+import sqlalchemy.exc as sa_exc
+import sqlalchemy.orm as sa_orm
+from flask import flash
+from flask import redirect
+from flask import render_template
+from flask import url_for
+from flask_login import current_user
+from markupsafe import Markup
+
+from pcapi.core.finance import models as finance_models
+from pcapi.core.history import api as history_api
+from pcapi.core.history import models as history_models
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.permissions import models as perm_models
+from pcapi.models import db
+from pcapi.repository.session_management import mark_transaction_as_invalid
+from pcapi.routes.backoffice import utils
+
+from . import forms
+
+
+non_payment_notices_blueprint = utils.child_backoffice_blueprint(
+    "non_payment_notices",
+    __name__,
+    url_prefix="/non_payment_notices",
```

Comment:
Par cohérence du BO : 
```suggestion
    url_prefix="/non-payment-notices",
```